sktime/transformations/series/scaledlogit.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implmenents the scaled logit tranformation."""
+
+__author__ = ["ltsaprounis"]
+__all__ = ["ScaledLogitTransformer"]
+
+from copy import deepcopy
+
+import numpy as np
+
+from sktime.transformations.base import BaseTransformer
+
+
+class ScaledLogitTransformer(BaseTransformer):
+    r"""Scaled logit transform.
+
+    Combined with a sktime.forecasting.compose.TransformedTargetForecaster it ensures
+    that the forecast stays between the specified bounds (lower_bound, upper_bound).
+    If both lower_bound and upper_bound are not None, a scaled logit transform is
+    applied. Otherwise, it applies a log transform variation that ensures the
+    resulting forecast is bounded as specified. The transform is applied to all
+    scalar elements of the input array individually.
+
+    Parameters
+    ----------
+    lower_bound : float
+        lower bound
```

Comment:
perhaps more precisely: of the log/logit function

Reply:
To be super precise it's the upper/lower bound of the inverse transform function

Reply:
well, the bounds of the transform function's *domain*, or the bounds of the inverse_transform function's *codomain*... whichever way you explain it, it should be clear *of what* this is a bound, you can also say "used in the scaled logit formula below" or similar